### ISSUE TITLE
/events frontmatter is messy

### ISSUE THREAD

**Issue by plhnk** [liferay-design/liferay.design#479]
```
draft: false #requried (true or false)

#Event Card Fields
author: 'fa_figma' #requried
title: 'Design at Scale' #requried
featuredImage: '/images/events/design-at-scale-madrid-2019.jpg' #requried
description: 'Madrid 🇪🇸' #required

#Event Details
startDateTime: '2019-02-12T00:00' #required (yyyy-mm-ddThh:mm)
endDateTime: ''

summary: 'Emiliano Cicero and Danny Saltaren shared their knowledge on how to build a component library in Figma, and on processes, tools and learnings respectively.'

locationName: 'Liferay Madrid, Spain'
locationURL: 'https://goo.gl/maps/JQZZugSQ5iG2'

heroImage: ''
bodyImage: '/images/events/design-at-scale-madrid-2019.jpg'
```

is too many things!

1. author — this doesn't make sense, an icon isnt an author??
1. description — same!? a city isnt a description
1. locationName — isn't this the same as city?
1. locationURL — am i trying to get a part-time job adding events??

**Comment by plhnk** [comment]
#477 should help some of this

**Comment by plhnk** [comment]
fixed in #474 — new frontmatter template for events is:

```
draft: false #required boolean (true or false)

#Event Card Fields
title: 
featuredImage: 
office: (needs to match one of the office id's in Offices.yaml)

#Event Details
startDateTime:  (yyyy-mm-ddThh:mm)
endDateTime: 

summary: 

heroImage: ''
bodyImage: ''
```
